eng/common/scripts/GeneralReleaseNotesLogic.ps1

```diff
@@ -0,0 +1,128 @@
+. (Join-Path $PSScriptRoot common.ps1)
+
+$PATTERN_REGEX = "^\[\w+(\.(?<SectionName>\w+))?\]:\s#\s\((?<Pattern>.*)\)"
+$CsvMetaData = Get-CSVMetadata
+
+function Get-PackagesInfoFromFile ($releaseNotesLocation) 
+{
+    $releaseNotesContent = Get-Content -Path $releaseNotesLocation
+    $checkLine = $False
+    $presentPkgInfo = @()
+
+    foreach ($line in $releaseNotesContent)
+    {
+        if ($line -eq "<!--")
```

Comment:
Should these be a little more lose? Maybe starts with instead of exact match, so that someone could put a comment after it on the same line.